code/modules/mob/living/carbon/human/human_update_icons.dm

```diff
@@ -431,12 +431,12 @@ There are several things that need to be remembered:
 		apply_overlay(LEGCUFF_LAYER)
 		throw_alert("legcuffed", /atom/movable/screen/alert/restrained/legcuffed, new_master = src.legcuffed)
 
-/proc/wear_female_version(t_color, icon, layer, type)
-	var/index = t_color
+/proc/wear_female_version(t_color, icon, layer, type, greyscale_colors)
+	var/index = t_color + greyscale_colors
 	var/icon/female_clothing_icon = GLOB.female_clothing_icons[index]
 	if(!female_clothing_icon) //Create standing/laying icons if they don't exist
-		generate_female_clothing(index,t_color,icon,type)
-	return mutable_appearance(GLOB.female_clothing_icons[t_color], layer = -layer)
+		generate_female_clothing(t_color, icon, type, greyscale_colors)
+	return mutable_appearance(GLOB.female_clothing_icons[t_color + greyscale_colors], layer = -layer)
```

Comment:
Why not use index here?

Reply:
It was because I had a bug with the indices, but it's refactored better now.